macaroons/service.go

```diff
@@ -300,6 +304,16 @@ func (svc *Service) GenerateNewRootKey() error {
 	return nil
 }
 
+// SetRootKey calls the underlying root key store's SetRootKey and returns the
+// result.
+func (svc *Service) SetRootKey(rootKey []byte) error {
+	if boltRKS, ok := svc.rks.(ExtendedRootKeyStore); ok {
+		return boltRKS.SetRootKey(rootKey)
+	}
+
+	return nil
```

Comment:
I'm not 100% sure it's a good idea to return `nil` here, since that means we didn't do anything. But looking at all the other methods here, that seems to be the pattern. So not going to block the PR on that, perhaps we should take another look at this whole service in another PR.

Reply:
Good point. Indeed I just went with the pattern that other methods were using.